packages/melange-recharts/melange-recharts.4.0.3/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "Melange bindings for recharts"
+description: "Melange bindings for recharts JavaScript library."
+maintainer: [
+  "Liubomyr Mykhalchenko <liubomyr.mykhalchenko@ahrefs.com>"
+]
+authors: [
+  "Liubomyr Mykhalchenko <liubomyr.mykhalchenko@ahrefs.com>"
+]
+license: "MIT"
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/melange-recharts/"
+bug-reports: "https://github.com/ahrefs/melange-recharts/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml"
+  "melange" {>= "3.0.0"}
+  "reason" {>= "3.10.0"}
+  "reason-react" {>= "0.14.0"}
+  "reason-react-ppx" {>= "0.14.0"}
+  "ocaml-lsp-server" {with-test}
```

Comment:
```suggestion
  "ocaml-lsp-server" {with-dev-setup}
```
Is it really a test dependency or just a dev setup one?

Reply:
I think we can remove this package from the deps directly. The reason why I'd rather not use `with-dev-setup` is that the current version of opam (2.1) doesn't support it, so there's no way to call `opam install --with-dev-setup` until 2.2 is out.

Reply:
I removed `ocaml-lsp-server` dependency